src/V3AstNodeOther.h

```diff
@@ -2851,6 +2851,16 @@ class AstCMethodHard final : public AstNodeStmt {
         statement(true);
         dtypeSetVoid();
     }
+    int instrCount() const override {
```

Comment:
Suggest move this to V3AstNodes.cpp.  It won't be inlined anyways (as is always called virtually) so shouldn't matter to performance.